contracts/FixedRateRewardsSource.sol

```diff
@@ -90,10 +90,6 @@ contract FixedRateRewardsSource is Governable, Initializable {
     function previewRewards() public view returns (uint256 rewardAmount) {
         RewardConfig memory _config = rewardConfig;
 
-        if (_config.lastCollect == 0) {
```

Comment:
Currently deploy script 10 deploys the contract and deploy script 12 sets the rewards per second. I would leave this check in since it is a sanity check and not very gas expensive one at that. 

With the current code of the contract, the contract needs to be deployed and funded in the correct order, or it will not function properly. Since one can: 
- run deploy script 10 that leaves the rewardsPerSecond to 0
- fund the contract
- call collectRewards and the function will probably revert, since previewRewards will return a too big of a number.

I think the correct functioning of the contract should be assured even if the deploy is not configured perfectly as expected. For that reason I would: 
- re-introduce the above removed lines of code and add comments why it is imperative that they are present
- remove the potentially confusing `_rewardsPerSecond` from the initialize function. Any deployment of this contract will suffer the same issues on initialisation of rewards source and start of emission. 
- add additional comments explicitly stating that the contract is basically paused until the `setRewardsPerSecond` isn't called with a non 0 value.
- currently it is also possible that the rewards source is already running (funded and non 0 rewardsPerSecond) and governor calls the `setRewardsPerSecond` with a 0 value. This would in effect pause the contract and remove any rewards accrued since the last collect. Is that intentional? If not change the `_setRewardsPerSecond` to:
  - either make it impossible to call the function with the 0 parameter 
  - or collect the rewards and send it to a configured target. And pause the rewards source by setting the rewards to 0

Reply:
Thanks for correcting me on first 2 points; you're right they are not an issue.

Reply:
I'm good with a change to rewards rate potentially changing how much users receive for the past, as well as the future. We'll probably usually couple it with a xOGN collect in most changes though.

Having a "pause" feature that stops rewards could be a useful thing.